frontend/src/config.ts

```diff
@@ -129,21 +129,38 @@ export const academicCalendars: { [season: Season]: SeasonCalendar } = {
       {
         name: 'Labor Day',
         start: [2024, 9, 2],
-        end: [2024, 9, 3],
+        end: [2024, 9, 2],
       },
       {
         name: 'October recess',
         start: [2024, 10, 16],
-        end: [2024, 10, 21],
+        end: [2024, 10, 20],
       },
       {
         name: 'November recess',
         start: [2024, 11, 23],
-        end: [2024, 12, 2],
+        end: [2024, 12, 1],
       },
     ],
     transfers: [{ date: [2024, 8, 30], day: 1 }],
   },
+  ['202501' as Season]: {
+    start: [2025, 1, 13],
+    end: [2025, 4, 25],
+    breaks: [
+      {
+        name: 'Martin Luther King Jr. Day',
+        start: [2025, 1, 20],
+        end: [2025, 1, 20],
+      },
+      {
+        name: 'Spring recess',
+        start: [2025, 3, 8],
+        end: [2025, 3, 23],
+      },
+    ],
+    transfers: [{ date: [2025, 1, 24], day: 1 }],
+  },
   // Add more entries above, but don't remove any
-  // Use zero-based indexing, i.e. "start" included, "end" excluded
+  // Use fully-closed indexing, i.e. "start" and "end" both included
```

Comment:
@benzuzu This is perhaps slightly confusing but the academic calendar itself is inconsistent. The dates for each break are left-right-exclusive ("Oct 17 – October recess begins, after last academic obligation"; "Oct 23 – classes resume"), while semesters are left-right-inclusive ("Apr 26 – classes end, 5:30pm"). In an effort to uniformize, I picked left-inclusive right-exclusive because that's the most typical boundary convention in JavaScript and other C-like languages. One has to do translations anyway even if both boundaries are inclusive.

Reply:
@Josh-Cena Are they not both left-right-inclusive now? I believe I changed it, otherwise labor day would not be working. 

`name: 'Labor Day',
        start: [2024, 9, 2],
        end: [2024, 9, 2],`

Reply:
Hmm, sure, no strong opinions. I'll soon refactor this to be dynamic instead of hardcoded anyway.